common/errcode.h

```diff
@@ -0,0 +1,11 @@
+#ifndef LIGHTNING_COMMON_ERRCODE_H
+#define LIGHTNING_COMMON_ERRCODE_H
+
+#include <inttypes.h>
+#include <stdint.h>
+
+typedef int32_t errcode_t;
```

Comment:
`#include<ccan/short_types/short_types.h>` instead, and `typedef s32 errcode_t;`, for consistency with the rest of the codebase.

Reply:
Done in fb8ab0aa.